woocommerce/woocommerce-android
# Feature/9598 add new profiler challenges step

Closes: #9598 


### Description

Adds a new profiler step asking the user about the challenges the face when setting up an online store. For now, this new step won't be accessible from anywhere in the app, until the store creation flow is completely revamped. 

- Clicking on "Skip" button will crash the app as the navigation to the next step is `TODO`
- Clicking on "Continue" will internally save the selected value but won't do anything else from UI perspective. 

### Testing instructions

- Apply the following patch
```diff
Subject: [PATCH] Add new store challenges fragment to navigation
---
Index: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/login/storecreation/countrypicker/CountryPickerFragment.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/login/storecreation/countrypicker/CountryPickerFragment.kt b/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/login/storecreation/countrypicker/CountryPickerFragment.kt
--- a/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/login/storecreation/countrypicker/CountryPickerFragment.kt	(revision 785d102cc447a89df7f53b455e5158004be3ed31)
+++ b/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/login/storecreation/countrypicker/CountryPickerFragment.kt	(date 1692108464578)
@@ -56,8 +56,11 @@
     }
 
     private fun navigateToInstallationStep() {
+//        findNavController().navigateSafely(
+//            CountryPickerFragmentDirections.actionCountryPickerFragmentToSummaryFragment()
+//        )
         findNavController().navigateSafely(
-            CountryPickerFragmentDirections.actionCountryPickerFragmentToSummaryFragment()
+            CountryPickerFragmentDirections.actionCountryPickerFragmentToStoreProfilerChallengesFragment()
         )
     }
 

```
- Trigger store creation flow and after the country picker screen you should see the following step. 
- Check the UI matches Figma design: 8sNT07kW3VEdnor4baI6ft-fi-387%3A43210

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/2663464/47352d4f-56f0-46c1-8914-547a1e3fb967
